Document/0x05b-Basic-Security_Testing.md

```diff
@@ -1244,13 +1244,43 @@ For enterprise tools, see the section "Static Source Code Analysis" in the chapt
 
 #### Dynamic Analysis
 
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1240>
+Dynamic Analysis tests the release version of an application, in a similar fashion to black box application testing. The way the code works should be inferred and looked at from a higher level, unlike static analysis, which takes the application's code and tries to analyze how the application will behave. This allows the tester to properly identify:
```

Comment:
Dynamic Analysis can test various versions of an app (UAT, pre-prod, prod etc.) and is also not directly linked to black box testing. Therefore I would change the following:

Dynamic Analysis tests the release version of an application, in a similar fashion to black box application testing. The way the code works should be inferred and looked at from a higher level, unlike static analysis, which takes the application's code and tries to analyze how the application will behave.

to:

Dynamic Analysis tests the mobile app by executing and running the app binary and analysing the workflows's for vulnerabilities. For example vulnerabilities regarding data storage might be sometimes hard to catch during static analysis, but in dynamic analysis you can easily spot what information is stored persistently and if the information is protected properly. Besides this, dynamic analysis allows the tester to properly identify: